traffic_portal/app/src/common/modules/form/iso/form.iso.tpl.html

```diff
@@ -147,8 +147,40 @@
+                <label class="control-label col-md-2 col-sm-2 col-xs-12">Management IP Netmask</label>
+                <div class="col-md-10 col-sm-10 col-xs-12">
+                    <input id="mgmtIpNetmask" name="mgmtIpNetmask" type="text" class="form-control" ng-model="iso.mgmtIpNetmask" ng-maxlength="50" autofocus>
+                    <small class="input-error" ng-show="hasPropertyError(isoForm.mgmtIpNetmask, 'maxlength')">Too Long</small>
+                    <span ng-show="hasError(isoForm.mgmtIpNetmask)" class="form-control-feedback"><i class="fa fa-times"></i></span>
+                </div>
+            </div>
+            <div class="form-group" ng-show="!isDHCP(iso)" ng-class="{'has-error': hasError(isoForm.mgmtIpGateway), 'has-feedback': hasError(isoForm.mgmtIpGateway)}">
+                <label class="control-label col-md-2 col-sm-2 col-xs-12">Management IP Gateway</label>
+                <div class="col-md-10 col-sm-10 col-xs-12">
+                    <input id="mgmtIpGateway" name="mgmtIpGateway" type="text" class="form-control" ng-model="iso.mgmtIpGateway" ng-maxlength="50" autofocus>
+                    <small class="input-error" ng-show="hasPropertyError(isoForm.mgmtIpGateway, 'maxlength')">Too Long</small>
+                    <span ng-show="hasError(isoForm.mgmtIpGateway)" class="form-control-feedback"><i class="fa fa-times"></i></span>
+                </div>
+            </div>
+            <div class="form-group" ng-show="!isDHCP(iso)" ng-class="{'has-error': hasError(isoForm.mgmtInterface), 'has-feedback': hasError(isoForm.mgmtInterface)}">
+                <label class="control-label col-md-2 col-sm-2 col-xs-12">Management Interface</label>
+                <div class="col-md-10 col-sm-10 col-xs-12">
+                    <input id="mgmtInterface" name="mgmtInterface" type="text" class="form-control" placeholder="Typical values are em3, eth3, etc. The name of the management interface. Required if Management IP Address is entered." ng-model="iso.mgmtInterface" ng-maxlength="50" autofocus>
+                    <small class="input-error" ng-show="hasPropertyError(isoForm.mgmtInterface, 'maxlength')">Too Long</small>
+                    <span ng-show="hasError(isoForm.mgmtInterface)" class="form-control-feedback"><i class="fa fa-times"></i></span>
+                </div>
+            </div>
             <div class="form-group" ng-class="{'has-error': hasError(isoForm.disk), 'has-feedback': hasError(isoForm.disk)}">
-                <label class="control-label col-md-2 col-sm-2 col-xs-12">Disk for OS Install</label>
+                <label class="control-label col-md-2 col-sm-2 col-xs-12">Disk for OS Install*</label>
                 <div class="col-md-10 col-sm-10 col-xs-12">
                     <input id="disk" name="disk" type="text" class="form-control" placeholder='Typical values are "sda"' ng-model="iso.disk" ng-maxlength="8" autofocus>
```

Comment:
make sure you add "required" to your input form element if disk is now required, otherwise TP code looks good

Reply:
Ok, I figured that was just for fields you wanted to validate.  Fixed.

Reply:
ok, i see your commit but make sure you add "required" to your input form element like this:

https://github.com/apache/incubator-trafficcontrol/blob/master/traffic_portal/app/src/common/modules/form/iso/form.iso.tpl.html#L51

Reply:
Those fields aren't required though?

Reply:
Oh, disk is though.  Took me a while